components/brave_savings/browser/perf_predictor_page_metrics_observer.cc

```diff
@@ -0,0 +1,92 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_savings/browser/perf_predictor_page_metrics_observer.h"
+
+#include "brave/browser/brave_browser_process_impl.h"
```

Comment:
including brave/browser and chrome/browser is a layering violation. brave/browser can depend on brave/components, but not the other way around